include/tallies/MeshTally.h

```diff
@@ -98,6 +102,12 @@ class MeshTally : public TallyBase
   /// OpenMC unstructured mesh instance for use with mesh tallies
   openmc::LibMesh * _mesh_template;
 
+  /// Whether the skinned mesh should be generated from a displaced mesh
+  const bool & _use_displaced;
+
+  /// Moose mesh
+  MooseMesh & getMooseMesh();
```

Comment:
This code is object-oriented. You automatically inherit everything (except `private` methods) from the base class. This may do weird things. Remove from this class and `CellTally`